engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/bv/MinValidatorForNumberTest.java

```diff
@@ -92,6 +98,18 @@ private void testDecimalMin(DecimalMin m, boolean inclusive) {
 		constraint.initialize( m );
 		testValidatorBigInteger( constraint, inclusive, false );
 
+		constraint = new DecimalMinValidatorForByte();
+		constraint.initialize( m );
+		testValidatorByte( constraint, inclusive, false );
+
+		constraint = new DecimalMinValidatorForShort();
+		constraint.initialize( m );
+		testValidatorShort( constraint, inclusive, false );
+
+		constraint = new DecimalMinValidatorForInteger();
+		constraint.initialize( m );
+		testValidatorInteger( constraint, inclusive, false );
```

Comment:
Do we have tests validating a decimal value for the new validators (in the case of `DecimalMin/Max`)?

Reply:
Both Decimal and regular Min/Max tests are using same asserts. So we should have the test from the original issue as well for Decimal validators.

Reply:
Yup, that's what worried me a bit: do they test a decimal limit properly?

Reply:
@marko-bekhta not sure if you have seen this one? If you think we are safe enough, I trust you, just wanted to share my doubts :).

Reply:
@gsmet  yes, thanks. I just got distracted with other work. I've just pushed an additional commit. We should squash it. I just made it as a separate so it'll be easier to see what I've added.